components/ContactForm.js

```diff
@@ -1,15 +1,60 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import ReCAPTCHA from "react-google-recaptcha";
+import { useRouter } from "next/router";
+import translations from "../translations";
 
 export default function ContactForm() {
+  let router = useRouter();
+  const locale = router.locale;
   const [isSuccess, setStatus] = useState(false);
   const [form, setForm] = useState({
     name: "",
     surname: "",
     email: "",
     message: "",
+    successMessage: "",
```

Comment:
skoro error masz po za tym to wydaje mi sie ze `isSuccess` tez powinno byc oddzielnie